src/blsprivatekey.cpp

```diff
@@ -41,7 +41,7 @@ BLSPrivateKey BLSPrivateKey::FromSeed(const uint8_t* seed, size_t seedLen) {
     bn_read_bin(*skBn, hash, BLSPrivateKey::PRIVATE_KEY_SIZE);
     bn_mod_basic(*skBn, *skBn, order);
 
-    BLSPrivateKey k = BLSPrivateKey();
+    BLSPrivateKey k;
```

Comment:
What's the difference here?

Reply:
Technically, you did a default-construction and assignment operations, so you had one unnecessary copy here. Practically, the compiler would have likely removed the unnecessary copy on it's own.